### PR TITLE
adding character limits to run task description

### PR DESCRIPTION
**Pull request by gautambaghel**
### What
adding character limits to run task description

### Why
char limit missing

### Screenshots

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [ ] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [ ] Description links to related pull requests or issues, if any.

#### Content
- [ ] Redirects have been added for moved, renamed, or deleted pages. This requires a separate PR in the [`terraform-website` repository](https://github.com/hashicorp/terraform-website) `redirects.next.js` file.
- [ ] API documentation and the API Changelog have been updated. 
- [ ] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [ ] Pages with related content are updated and link to this content when appropriate.
- [ ] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [ ] New pages have metadata (page name and description) at the top.
- [ ] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [ ] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [ ] UI elements (button names, page names, etc.) are bolded.
- [ ] The Vercel website preview successfully deployed.

#### Reviews
- [ ] I or someone else reviewed the content for technical accuracy.
- [ ] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

**Comment by gautambaghel** [comment]
Hi @mgarrell777 ! Can you help me out with this PR? Thanks! Ping me internally if you need additional context

**Review by glennsarti — APPROVED** [review]
Confirmed within Atlas that indeed the limit is 300 chars long.

File: website/docs/cloud-docs/api-docs/run-tasks/run-tasks.mdx [hunk 1]
```diff
@@ -75,7 +75,7 @@ Properties without a default value are required unless otherwise specified.
 | `data.type`                   | string |         | Must be `"tasks"`.                                                                            |
 | `data.attributes.name`        | string |         | The name of the task. Can include letters, numbers, `-`, and `_`.                             |
 | `data.attributes.url`         | string |         | URL to send a run task payload.                                                               |
-| `data.attributes.description` | string |         | The description of the run task. Can include spaces, letters, numbers, and special characters |
+| `data.attributes.description` | string |         | The description of the run task. Can include spaces, letters, numbers, and special characters up to 300 characters. |
```
Comment (mgarrell777) [inline website/docs/cloud-docs/api-docs/run-tasks/run-tasks.mdx:78]:
```suggestion
| `data.attributes.description` | string |         | The description of the run task. Can be up to 300 characters long including spaces, letters, numbers, and special characters. |
```

Suggest changing the character description to "Can be up to 300 characters long including spaces, letters, numbers, and special characters" for clarity.

File: website/docs/cloud-docs/api-docs/run-tasks/run-tasks.mdx [hunk 2]
```diff
@@ -314,7 +314,7 @@ Properties without a default value are required unless otherwise specified.
 | `data.type`                   | string |                  | Must be `"tasks"`.                                                                            |
 | `data.attributes.name`        | string | (previous value) | The name of the run task. Can include letters, numbers, `-`, and `_`.                         |
 | `data.attributes.url`         | string | (previous value) | URL to send a run task payload.                                                               |
-| `data.attributes.description` | string |                  | The description of the run task. Can include spaces, letters, numbers, and special characters |
+| `data.attributes.description` | string |                  | The description of the run task. Can include spaces, letters, numbers, and special characters up to 300 characters. |
```
Comment (mgarrell777) [inline website/docs/cloud-docs/api-docs/run-tasks/run-tasks.mdx:317]:
```suggestion
| `data.attributes.description` | string |                  | The description of the run task. Can be up to 300 characters long including spaces, letters, numbers, and special characters. |
```
Suggest changing the character description to "Can be up to 300 characters long including spaces, letters, numbers, and special characters" for clarity.

**Review by mgarrell777 — APPROVED** [review]
@gautambaghel Thanks for your contribution. I have a quick wording change suggestion to add clarity, but otherwise looks good to me.

**Review by mgarrell777 — APPROVED** [review]
@gautambaghel Looks good to me. Thanks for your efforts.

**Comment by mgarrell777** [comment]
> @mgarrell777 -- Let's merge this :)

Done and done :)

**Pull request merged**